cmd/libguestfs/BUILD.bazel

```diff
@@ -27,6 +27,7 @@ container_image(
         "//:get-version",
     ],
     tars = [
+        "//:passwd-tar",
```

Comment:
The initial passwd entry fot the `1001` user was:
```
bash-4.4$ cat /etc/passwd
...
1001:x:1001:0:1001 user:/usr/local/lib/guestfs/appliance:/sbin/nologin
```
With this change:
```
bash-4.4$ cat /etc/passwd 
nonroot-user:x:1001:1001:user:/home/nonroot-user:/bin/bash
```

Reply:
@vasiliy-ul nice catch! I don't know what is happening, maybe bazel doesn't properly build tar rule :/ but this is my understanding too.

Reply:
I think bazel just picks the first occurrence of `/etc/passwd` from the tarballs and overwrites the file in the base image. So without having `"//:passwd-tar"` here, it will come from `"//rpm:libguestfs-tools"`.